modules/date/date-parser-parser.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2013 BalaBit IT Ltd, Budapest, Hungary
+ * Copyright (c) 2013 Gergely Nagy <algernon@balabit.hu>
```

Comment:
Again, the copyright could use an update. Both the year, and your name should be there, instead of mine. (Goes for all other similar occurrences).